app/Controller/Component/RestResponseComponent.php

```diff
@@ -382,6 +382,7 @@ public function saveFailResponse($controller, $action, $id = false, $validationE
         if (isset($this->__convertActionToMessage[$controller][$action['action']])) {
             $stringifiedAction = $this->__convertActionToMessage[$controller][$action['action']];
         }
+        $response['saved'] = false;
```

Comment:
Revert this

Reply:
We use it for ajax queries, the restresponse component however is used for API queries. It's inconsistent with the rest of the API.